feature-group-definitions/web-animations.yml

```diff
@@ -44,7 +55,8 @@ compat_features:
   - api.KeyframeEffect.KeyframeEffect
   - api.KeyframeEffect.composite
   - api.KeyframeEffect.getKeyframes
-  - api.KeyframeEffect.iterationComposite
+  # TODO: iterationComposite is a later addition and needs a better home than this feature
```

Comment:
Looking at the spec, this indeed only appears in the level 2 spec, not level 1. So if this feature is focused on level 1 only, then maybe we need a second feature file that contains level 1 + whatever was added in level 2, like iterationComposite.